dev/listarith.tex

```diff
@@ -161,14 +161,14 @@ \section{Determining the Type of Plists}
 **
 **  TypePlist works with KTnumPlist to determine the type of a plain list
 **  Considerable care is needed to deal with self-referential lists. This is
-**  basically achieved with the TESTING flag in the Tnum. This must be set in
+**  basically achieved with the OF_TESTING flag in the Tnum. This must be set in
```

Comment:
Does the `OF` in `OF_TESTING` stand for `object flag`? I keep trying to pronounce it in my head as the English with "of", resulting in "of testing", which is weird.

I think it would be nice if you explicitly wrote somewhere that `OF_` things are object flags.

Reply:
Yes it is meant to stand for "object flags". We already have three related functions: `TEST_OBJ_FLAG`, `SET_OBJ_FLAG`, `CLEAR_OBJ_FLAG`. So of course we could also use `OBJ_FLAG_TESTING` or `TESTING_OBJ_FLAG`, but that seemed a bit longish. Anyway, I'd be happy to change this, too.

Reply:
I'm not necessarily asking you to change it; `OBJ_FLAG_TESTING` is certainly longish. Another suggestion is `O_F_TESTING`, which is a compromise in terms of length, and avoids suggesting that `OF` is a word, but is also uglyish...

Perhaps @ChrisJefferson has an opinion.

Reply:
My personal preference is `OF_TESTING` > `OBJ_FLAG_TESTING` > `O_F_TESTING`, although I do see the potential for confusion with `OF_` and the word "of". I am happy to settle on `OBJ_FLAG_TESTING` then. But sure, let's see if @ChrisJefferson has thoughts. In the end, though, we can quite easily repaint the bikeshed later on ;-)

Reply:
I've now switched this PR to `OBJ_FLAG_TESTING` resp. `OBJ_FLAG_TESTED`.

Reply:
I think I also have slight preference against `OF`, the new words are perfectly fine :)